presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerRecordSet.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.sqlserver;
+
+import com.facebook.presto.plugin.jdbc.JdbcClient;
+import com.facebook.presto.plugin.jdbc.JdbcColumnHandle;
+import com.facebook.presto.plugin.jdbc.JdbcRecordSet;
+import com.facebook.presto.plugin.jdbc.JdbcSplit;
+import com.facebook.presto.spi.RecordCursor;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+class SqlServerRecordSet
```

Comment:
Introduction of this class & plumbing around it should also be separate commit from data type changes.